doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+.. _detailed-release-procedure:
+
+Release: recipes runs and comparison
+====================================
+
+The release procedure for ESMValTool is a fairly involved process (at the moment), so it
+is important to be very well organized and to have documented each procedural steps, so that
+the next release manager can follow said steps, and finalize the release without any delays.
+
+The workflow below assumes an ESMValCore release candidate, or a completed stable release, have been released
+and deployed on conda-forge and PyPI; it also assumes the release manager has access to accounts on DKRZ/Levante.
+
+Below is a list of steps that the release manager, together with the previous release manager, should go through before the actual release;
+these include testing the new code by running all available recipes in yhe `main` branch, and comparing the output against
+the previous release.
+
+Open an issue on GitHub
+-----------------------
+
+First, open an issue on GitHub where the testing workflow before the release is documented (see example https://github.com/ESMValGroup/ESMValTool/issues/2881).
+Name it something relevant like "Recipe testing and comparison for release 2.x.x", and populate the issue description with information
+about where the testing is taking place, what tools are used, and what versions, here are some suggestions:
+
+
+Where the work is done e.g. work done on DKRZ/Levante:
+
+- submit files in `/home/b/b382109/submit` (for v2.7.0)
+- output in `/scratch/b/b382109/esmvaltool_output` (for v2.7.0)
+
+We should document various utilities' versions so that the work can be reproduced in case there
+is an issue, or release work needs to be picked up mid-release by another release manager:
+
+- documenting `conda`/`mamba` versions:
+
+.. code-block:: bash
+
+  mamba --version
+
+- documenting `git` branch and its state:
+
+.. code-block:: bash
+
+  git status
+
+Furthermore, the runtime environment needs to be documented:
+
+.. code-block:: bash
+
+  mamba env create -n tool2xxTest -f environment.yml
+  conda activate tool2xxTest
```

Comment:
This looks like part of the installation instructions. It will be easier to maintain this documentation if it just includes a link instead of some duplicate text.

Reply:
good point! Let me see how I can wedge that in

Reply:
I have actually just removed it, the RM should really know how to install by now :laughing: